### ISSUE TITLE
Could not work with a TypeError.

### ISSUE THREAD

**Issue by devchick** [bouzuya/hubot-lgtm#1]
I got an error as below.

```
ERROR TypeError: Object 1 has no method 'then'
  at TextListener.callback (/path/to/hubot/node_modules/hubot-lgtm/src/scripts/lgtm.coffee:47:5, <js>:46:26)
```

**Comment by bouzuya** [comment]
`lgtm.coffee` is written for `coffee-script >= 1.7`, but `hubot` uses `coffee-script@1.6.2` as the default. You can use `/lib/scripts/lgtm.js`.

I recommend you to use `hubot-lgtm` as an external script.

**Comment by bouzuya** [comment]
oh...

https://github.com/bouzuya/hubot-lgtm/blob/master/index.js#L4
> robot.load(path.join(__dirname, '/src/scripts/'));

I will try to fix it.

**Comment by bouzuya** [comment]
I fixed it in `2.0.2`. https://github.com/bouzuya/hubot-lgtm/releases/tag/2.0.2

**Comment by devchick** [comment]
Thanks! I have resolved my isuue!
